vendor/tracing/zero/destinations.ml

```diff
@@ -32,7 +32,69 @@ let direct_file_destination ?(buffer_size = 4096 * 16) ~filename () =
   (module Dest : Writer_intf.Destination)
 ;;
 
-let file_destination ~filename () = direct_file_destination ~filename ()
+let compressed_file_destination ?(buffer_size = 4096 * 16) ~filename () =
```

Comment:
Hmm, is there a semantic reason why this is written as 4 KiB * 16? Would `64 * 1024` or `65536` make more sense here?